src/main/proto/tech/figure/servicing/v1beta1/loan_state.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+package tech.figure.servicing.v1beta1;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+import "tech/figure/util/v1beta1/person.proto";
+import "tech/figure/util/v1beta1/types.proto";
+import "validate/validate.proto";
+
+/**
+Aggregation of loan state across a structure (group) of loans
+ */
+message LoanStructureState {
+  string             marker_address = 1; // Provenance Blockchain Address for the Marker representing this loan structure
+  repeated LoanState loan_states    = 2; // Loan state data for loans included in the structure
+}
+
+/*
+Loan state data (servicing data) at a point in time
+ */
+message LoanState {
+  tech.figure.util.UUID     loan_id               = 1 [(validate.rules).message.required = true];                                             // Loan identifier for which this state is calculated
+  google.protobuf.Timestamp effective_time        = 2 [(validate.rules).timestamp.required = true, (validate.rules).timestamp.lt_now = true]; // Timestamp when this state was produced
+  string                    loan_type             = 3 [(validate.rules).string.min_len = 1];                                                  // Type of loan (e.g. MORTGAGE, HELOC, PERSONAL_LOAN)
```

Comment:
Should loan_type also be required or are there cases where a loan type is unnecessary or not applicable?